### PR TITLE
Update doc/dokuvimki.txt

### PR DESCRIPTION
**Pull request by oscarmlage**
* Changed links s/chimeric/kynan
* Fixed issus typo

### CODE REVIEW

File: doc/dokuvimki.txt [hunk 1]
```diff
@@ -188,7 +188,7 @@ BUGS                                                          *dokuvimki-bugs*
 
 Please use the official bug tracker for feature requests of bug reports:
 >
-    http://github.com/chimeric/dokuvimki/issus
+    http://github.com/chimeric/kynan/issues
```
Comment (kynan) [inline doc/dokuvimki.txt]:
```suggestion
    http://github.com/kynan/dokuvimki/issues
```

**Comment by kynan** [comment]
@oscarmlage could you please address the outstanding issues?

**Comment by oscarmlage** [comment]
> @oscarmlage could you please address the outstanding issues?

Done, sorry for the delay!

File: doc/dokuvimki.txt [hunk 2]
```diff
@@ -24,7 +24,7 @@ UPDATES                                                    *dokuvimki-updates*
 
 The official releases can be found at:
 
-    git: http://github.com/chimeric/dokuvimki
+    git: http://github.com/kynan/dokuvimki
```
Comment (kynan) [inline doc/dokuvimki.txt]:
```suggestion
    git: https://github.com/kynan/dokuvimki
```

File: doc/dokuvimki.txt [hunk 3]
```diff
@@ -188,7 +188,7 @@ BUGS                                                          *dokuvimki-bugs*
 
 Please use the official bug tracker for feature requests of bug reports:
 >
-    http://github.com/chimeric/dokuvimki/issus
+    http://github.com/kynan/dokuvimki/issues
```
Comment (kynan) [inline doc/dokuvimki.txt]:
```suggestion
    https://github.com/kynan/dokuvimki/issues
```

**Comment by kynan** [comment]
Thanks!

**Pull request merged**